sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ### Bugs Fixed
 - Long-Running operation rehydration has been patched to stop throwing a `NullReferenceException`. Issue [24692](https://github.com/Azure/azure-sdk-for-net/issues/24692).
+- TextAnalyticsAudience has been added to allow the user to select the Azure cloud where the resource is located. Issue [18520])https://github.com/Azure/azure-sdk-for-net/issues/18520).
```

Comment:
nit: formatting typo when writing the link. It should be `[18520](...` instead of `[18520])...`.